server.js

```diff
@@ -12,7 +12,7 @@ app.use(express.static('build'));
 app.use(bodyParser.json());
 
 app.get("/", function(req, res){
-  res.sendFile(__dirname + "/build/index.html")
+  res.sendFile(__dirname +"build/index.html")
```

Comment:
I think we need "/" here
https://nodejs.org/docs/latest/api/globals.html#globals_dirname